client/pom.xml

```diff
@@ -11,17 +11,23 @@
     <name>tds-student-service-client</name>
     <description>The client classes for interacting with the student service</description>
 
+    <parent>
+        <groupId>org.opentestsystem.student</groupId>
+        <artifactId>tds-student-service</artifactId>
```

Comment:
I didn't do that originally because of the existing student application.  Our groups are different so we can do it.  Just wasn't sure if it would be confusing.


```
	<groupId>org.opentestsystem.delivery</groupId>
	<artifactId>student</artifactId>
	<name>student</name>
	<packaging>war</packaging>
	<version>2.1.1-SNAPSHOT</version>
```